src/Raven.Studio/wwwroot/Content/css/pages/time-series-edit.less

```diff
@@ -1,4 +1,4 @@
-.edit-time-series {
+.edit-time-series {    
```

Comment:
please avoid adding spaces at the end of line. 

Maybe you should use shortcuts for deleting entire lines like ctrl + shift + d.